.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+
+[vcbuild.bat]
+end_of_line = crlf
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
+
+[{lib,src,test}/**.js]
+indent_style = space
+indent_size = 2
+
+[src/{**.h,**.cc}]
```

Comment:
I think it's fine to keep them separated (easier to read and to make changes to only JS or C++ rules in the future)